bindings/python/src/peer_info.cpp

```diff
@@ -82,7 +83,7 @@ void bind_peer_info()
         .def_readonly("download_queue_length", &peer_info::download_queue_length)
         .def_readonly("upload_queue_length", &peer_info::upload_queue_length)
         .def_readonly("failcount", &peer_info::failcount)
-        .def_readonly("downloading_piece_index", &peer_info::downloading_piece_index)
+        .add_property("downloading_piece_index", make_getter(&peer_info::downloading_piece_index, by_value()))
```

Comment:
I did not try to understand which was causing the issue.
I will add
https://github.com/Falcosc/libtorrent/blob/bd72ea64c508e2406d5e6ed1abe2f29b76f1341b/bindings/python/src/peer_info.cpp#L87
https://github.com/arvidn/libtorrent/blob/4f89b397b1d53c9043f28b04757950399111400b/bindings/python/src/torrent_handle.cpp#L476
https://github.com/arvidn/libtorrent/blob/8800cae5854cd6993664128570ea8a8f1a9efb44/bindings/python/src/torrent_info.cpp#L221

And all piece and block index in the alert binding as well?
https://github.com/arvidn/libtorrent/blob/13f9c110789df67c10f223d02eb9ed8633f6b7c7/bindings/python/src/alert.cpp